section1/reflection.md

```diff
@@ -1,19 +1,33 @@
 ## Section 1 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+* I am usually a little nervous about asking questions but I will still ask them. I tend to be somewhere in the middle.
```

Comment:
Same! Things ramp up fast in Mod 1, so I recommend being conscious of if you're still in the middle and whether you'd like to ask more questions.